### PR TITLE
Adds Issue Templates to Issues

### PR DESCRIPTION
**Pull request by jlaura**
## Description

PR adds four issue templates for helping to categorize issues. This PR makes a distinction between bugs and features to assist in watching version numbers.

## Related Issue

#661 
## Motivation and Context

We are seeking to provide additional support to contributors opening issues.

## How Has This Been Tested?

No testing is required as this is not a code change.
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Review by jessemapel — DISMISSED** [review]
Looks good to me, I'd like another set of eyes to check it before we merge this.

File: .github/ISSUE_TEMPLATE/3_Support_question.md
Comment (SgStapleton) [inline .github/ISSUE_TEMPLATE/3_Support_question.md]: "For questions about using ISIS3 or other ASC software, ..."

**Review by SgStapleton — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/3_Support_question.md
Comment (SgStapleton) [reply .github/ISSUE_TEMPLATE/3_Support_question.md]: Or something of the like. Not entirely sure what was meant here.

**Comment by jlaura** [comment]
@SgStapleton Does this look okay? If so, lets wait to merge until discourse is live. Then we can flip the switch.

@scsides Do you know if issue templates will interfere with the migration?

**Comment by scsides** [comment]
I have no idea at this point.

**Comment by jessemapel** [comment]
@jlaura @scsides Can we merge this and then if they are an issue for the transition, disable them until that is done?

**Comment by jlaura** [comment]
@jessemapel Sure on the RedMine side. It looks like we are almost live on astrodiscuss.usgs.gov. I would like to wait for that to go live since the question templates asks users to use that. I do not want to have questions from contributors in limbo.

**Review by SgStapleton — APPROVED** [review]

**Comment by jlaura** [comment]
@all safe to merge?

**Comment by jlaura** [comment]
Once #690 is merged, we should be able to merge this.

**Pull request merged**
